client/lib/deploy.rs

```diff
@@ -251,6 +252,11 @@ impl DeployExt for Deploy {
         const TRANSFER_ARG_TARGET: &str = "target";
         const TRANSFER_ARG_ID: &str = "id";
 
+        let amount = U512::from_dec_str(amount)
+            .map_err(|err| Error::FailedToParseUint("amount", UIntParseError::FromDecStr(err)))?;
```

Comment:
Could it be that we can use the `const TRANSFER_ARG_AMOUNT` here instead of hardcoding the `"amount"` string?

Reply:
Yes that makes sense.